jwst/associations/lib/rules_level3.py

```diff
@@ -474,6 +474,10 @@ def __init__(self, *args, **kwargs):
         # Check and continue initialization.
         super(Asn_Lv3NRCCoronImage, self).__init__(*args, **kwargs)
 
+    @property
```

Comment:
For reviewers, this just adds an override for the regular "dms_product_name" function, to be used by only this particular ASN class.